src/optimizers.py

```diff
@@ -5,6 +5,8 @@
 from network import *
 import numpy as np
 
+from network import Network
```

Comment:
```suggestion
from network import *
```

Se no ti dà quell'errore sull'import, tipo "partially initialized qualcosa"